mesos/framework/cluster.go

```diff
@@ -56,6 +57,7 @@ func (c *Cluster) Add(task Task) {
 	c.taskLock.Lock()
 	defer c.taskLock.Unlock()
 
+	fmt.Println(task.Data())
```

Comment:
We _really_ need a central logger that produces to kafka. We might even just add that to elodina/go-mesos-utils this way it can search the master state json (or such) for the kafka brokers we know and expect as brokers for us to write the logs (and metrics) too. If not found then we can force then to have to supply one on the command line or such.